### PR TITLE
feat: move 1on1 conversation messages [WPB-6246]

### PR DESCRIPTION
**Pull request by markusfassbender**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6246" title="WPB-6246" target="_blank">Sub-taskWPB-6246</a>  [iOS] move 1:1 conversation messages to new mls conversation
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

See ticket.

Open questions:
- Should the proteus conversation be deleted afterwards?

### Solutions

Copy messages from proteus to mls conversation

#### Test Coverage (Optional)

- [x] I have added automated test to this contribution

#### How to Test

Testing is quite difficult, because this is an edge case that should only work as safety network if all other migrations before have failed.

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: wire-ios-data-model/Tests/OneOnOne/OneOnOneMigratorTests.swift [hunk 1]
```diff
@@ -99,59 +91,154 @@ final class OneOnOneMigratorTests: ZMBaseManagedObjectTest {
...
         // Mock
         _ = MockActionHandler<SyncMLSOneToOneConversationAction>(
             result: .success(mlsGroupID),
             context: uiMOC.notificationContext
         )
 
-        mlsService.conversationExistsGroupID_MockValue = true
-
         // When
+        let mockMessage = "Hello World!"
```
Comment (johnxnguyen) [inline wire-ios-data-model/Tests/OneOnOne/OneOnOneMigratorTests.swift]: What do you think about appending a few more messages, and possibly different types of messages?

File: wire-ios-data-model/Tests/Source/Model/User/UserImageLocalCacheTests.swift [hunk 2]
```diff
@@ -17,33 +17,42 @@
 // 
 
 import Foundation
-import WireDataModel
+@testable import WireDataModel
 
-extension ZMUser {
+final class UserImageLocalCacheTests: XCTestCase {
```
Comment (markusfassbender) [inline wire-ios-data-model/Tests/Source/Model/User/UserImageLocalCacheTests.swift:22]: I just touched this test because it seems to be flaky on my branch 🤷‍♂️

**Review by johnxnguyen — APPROVED** [review]

**Review by agisilaos — APPROVED** [review]
